.github/workflows/test-go1.10.yaml

```diff
@@ -1,18 +1,18 @@
-name: Tests
+name: Tests go1.10
 on:
   pull_request:
     branches: [ "master", "release/**" ]
 jobs:
-  unit-tests:
+  unit-tests-1_10:
```

Comment:
Each workflow has private identifiers, you can just keep this at `unit-tests` without ill effects.

Reply:
That's what I did initially but iirc it looked not super nice in the web UI, it was a bit confusing what was running where etc.